### PR TITLE
Update manual to use new role names

### PR DESCRIPTION
**Pull request by tbrisker**
Anonymous -> Default Role
Default User -> /dev/null

### CODE REVIEW

File: _includes/manuals/1.12/4.1.2_roles_and_permissions.md [hunk 1]
```diff
@@ -2,23 +2,15 @@ A user's access to the features of Foreman are constrained by the **permissions*
 
 Note: a user with global admin enabled is not restricted by the authorization system. This is the default for installations that do not have :login:true in _config/settings.yml_.
 
-A logged in user will be granted the **Anonymous** role plus one or more additional roles. The permissions and filters associated with these roles are aggregated and determine the final permission set.
+A logged in user will be granted the **Default Role** role plus one or more additional roles. The permissions and filters associated with these roles are aggregated and determine the final permission set.
```
Comment (domcleal) [inline _includes/manuals/1.12/4.1.2_roles_and_permissions.md]: `Default role` (lower case 'r')

File: _includes/manuals/1.12/4.1.2_roles_and_permissions.md [hunk 2]
```diff
@@ -2,23 +2,15 @@ A user's access to the features of Foreman are constrained by the **permissions*
...
-There are two builtin system roles
-
-
-1. Anonymous: This is a set of permissions that every user at your installation will be granted, irrespective of any other roles that they have.
-
-
-2. Default user: When a new Role is created this set of permissions are used as the template for the Role. The name is somewhat misleading but basically an ordinary default user who was assigned this Role would have these permissions set.
-
-
+There is one built-in system role, 'Default Role'. This is a set of permissions that every user at your installation will be granted, irrespective of any other roles that they have.
```
Comment (domcleal) [inline _includes/manuals/1.12/4.1.2_roles_and_permissions.md]: 1) ditto re case.
2) "irrespective of" => "in addition to"
3) I'd perhaps leave out "at your installation".

**Comment by tbrisker** [comment]
@domcleal fixed

**Comment by domcleal** [comment]
Thanks @tbrisker!

**Pull request merged**